browser-test/src/admin/admin_publish.test.ts

```diff
@@ -70,6 +70,63 @@ test.describe(
   },
 )
 
+test.describe(
+  'publishing all programs with disabled programs feature flag on',
+  {tag: ['@uses-fixtures']},
```

Comment:
**thought:** @gwendolyngoetz curious if you think its worth adding a tag for this feature or if that's more for larger flags like the northstar work?

Reply:
There's no harm in adding tags from playwrights perspective. Manage them may become a chore. This is what I'm thinking.

Good candidates for tags (conceptually, not necessarily the names):
- Roles (admin, applicant, etc)
- Environment (local only, real staging environment, etc)
- In development (like northstar)
- Status indicator (slow, flaky, etc)

Poor candidates for tags:
- Super fine grained (admin add question page, settings page)
- Only going to be used by a few things
- Short lived (feature flag 1, feature flag 2)

In this case perhaps we go with `in-development` and use that for longer running feature flag work.

Reply:
That sounds great, thank you!!